docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst

```diff
@@ -229,6 +233,11 @@ Including,
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * :ref:`Pregnancy model <other_models_pregnancy>`
+
+.. todo::
+
+  Clarify with software engineers if updates were made to the pregnancy model to accomodate the accrual of maternal disorders YLDs (a timestep long post-maternal disorders state in which anemia YLDs are not accrued?). If so, then add these updates to the pregnancy model documentation.
```

Comment:
We did do this. We have two "prepostpartum" states `maternal_disorder` and `no_maternal_disorder`, former being simulants with non-fatal maternal disorders and YLDs and the latter is "normal" no disorder.

Reply:
Awesome, thank you for clarifying. I will add this to the docs!